projects/extension/sql/idempotent/013-loading.sql

```diff
@@ -0,0 +1,83 @@
+-------------------------------------------------------------------------------
+-- loading_row
+create or replace function ai.loading_row
+( column_name pg_catalog.text)
+returns pg_catalog.jsonb
+as $func$
+    select json_object
+    ( 'implementation': 'row'
+    , 'config_type': 'loading'
+    , 'column_name': column_name
+    )
+$func$ language sql immutable security invoker
+set search_path to pg_catalog, pg_temp
+;
+
+-------------------------------------------------------------------------------
+-- loading_document
+create or replace function ai.loading_document
```

Comment:
IMO, `loading_uri` would be a more appropriate name. It describes "where" we are loading from.